src/main/resources/templates/data_request_access.html

```diff
@@ -8,15 +8,54 @@
     <body class="mybg loginbackground">
         <div th:replace="fragments/dashboard_navigation :: dashboardnav">&nbsp;</div>
         <p>&nbsp;</p>
-        <div class="container">
-            <div class="row boxes responsive">
+        <p>&nbsp;</p>
+        <p>&nbsp;</p>
+        <p>&nbsp;</p>
+        <div class="container ">
+
+            <div class="row boxes responsive dashpanel">
                 <h2 class="form" th:text="#{dataset.request.sent}">Your requested has been submit.</h2>
                 <p th:text="#{dataset.request.info}">We have forwarded your request to the data’s owner for review.</p>
                 <p>Please contact the <strong>Data Owner: </strong>&lt;<span th:text="${ownerName}">Data Owner</span>&gt; <strong>Email: </strong>&lt;<span th:text="${ownerEmail}">team-owner@email</span>&gt; for any issues.</p>
                 <p>&nbsp;</p>
                 <p>&nbsp;</p>
             </div> <!-- row -->
+            <div class="cc-wraper">
+                <div class="cc-contactpop fixed">
+                    <h1>Do you need Help? <span>Get in touch.</span></h1>
+                    <div class="renseignement">
+                        <i class="fa fa-times"></i>
+                        <p>We will get back to you as soon as possible. </p>
+                    </div>
+                    <form>
+                        <ul>
+                            <li><i class="fa fa-phone"></i> <span> Phone: </span>+65 6601 1056</li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)